### ISSUE TITLE
cannot end process: database is locked

### ISSUE THREAD

**Issue by p4u** [vocdoni/vocdoni-node#634]
Docker compose integration test produced the following output (not always). 

```
2022-10-18T11:04:06Z	INFO	rpcclient/api.go:839	votes submited! took 2.149642902s
2022-10-18T11:04:06Z	INFO	rpcclient/api.go:842	waiting for votes to be validated...
2022-10-18T11:04:24Z	INFO	vochaintest/vochaintest.go:1063	gateway 0 http://gateway0:9090/dvote has ended its job
2022-10-18T11:04:24Z	INFO	vochaintest/vochaintest.go:1063	gateway 1 http://gateway0:9090/dvote has ended its job
2022-10-18T11:04:24Z	INFO	vochaintest/vochaintest.go:1071	ending process in order to fetch the results
2022-10-18T11:04:24Z	INFO	rpcclient/util.go:21	waiting for block 9...
2022-10-18T11:04:30Z	INFO	rpcclient/util.go:33	remaining blocks: 1
2022-10-18T11:04:35Z	WARN	vochaintest/vochaintest.go:324	ensureProcessEnded: cannot force end process: cannot get process 5734e11003c5074fcaacb8b01850539eae7e9fee8dc94549db96030400000000 info: cannot getProcessInfo: cannot get process info: database is locked
2022-10-18T11:04:35Z	INFO	vochaintest/vochaintest.go:326	ensureProcessEnded: got process 5734e11003c5074fcaacb8b01850539eae7e9fee8dc94549db96030400000000 info <nil>
Error: Process completed with exit code 2.
```

**Comment by p4u** [comment]
related to https://github.com/vocdoni/vocdoni-node/pull/626 ?

**Comment by altergui** [comment]
nope. that bug is fixed in master, but you're using release-azeno branch which didn't yet merge all the CI fixes from #608
i just checked and there are no conflicts, you can just `git merge master` and be happy
